corehq/apps/export/models/new.py

```diff
@@ -1989,11 +1985,11 @@ def get_latest_export_schema(domain, app_id, case_type):
 
     @classmethod
     def _process_app_build(cls, current_schema, app, case_type):
-        case_property_mapping = get_case_properties(
-            app,
-            [case_type],
+        case_property_mapping = {case_type: sorted(get_all_case_properties_for_case_type(
+            app.domain,
+            case_type,
             include_parent_properties=False
-        )
+        ))}
```

Comment:
I think this will break the schema generation for app builds since it's no longer looking at the app passed in to the function.

My understanding of how this used to work:
1. process all 'latest' apps in domain
2. process all app builds in domain

Removing (2) will mean that any case properties that have been removed from the current app will no longer be present in the schema.

Reply:
I do keep that list of app builds when `only_process_current_builds` is `False` , I can probably pass that parameter in, and use this workaround when on when it is false

Reply:
Ended up just leaving `_process_app_build` as it was before since it already pretty much does the same thing as `get_case_properties_in_domain`. Now the only difference in this file is that `generate_schema_from_builds` does not pull in current apps not directly referenced, but the `ParentCasePropertyBuilder` already does that, so now its runs in N time with apps instead of N^2, in addition to the `name_formatter` changes